src/components/Navbar/Navbar.css

```diff
@@ -69,6 +70,12 @@
   width: 30%;
 }
 
+#logout-btn {
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+  cursor: pointer;
+}
+
 @media screen and (max-width: 800px) {
```

Comment:
Read more about responsive design and common breakpoints. We should avoid using non-standard values. Please check this https://blog.logrocket.com/css-breakpoints-responsive-design. Thanks.